<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>401</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelcontents">
       <property name="font">
        <font>
         <pointsize>24</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>ShotInfo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTreeWidget" name="tree_shot">
       <column>
        <property name="text">
         <string>shot</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>mov</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>hip</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>nk</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="accessibleName">
        <string notr="true"/>
       </property>
       <property name="value">
        <number>24</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="textVisible">
        <bool>true</bool>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="invertedAppearance">
        <bool>false</bool>
       </property>
       <property name="textDirection">
        <enum>QProgressBar::TopToBottom</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_run">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
